packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeDetailsItem/MT5TradeDetailsItem.tsx

```diff
@@ -25,15 +25,17 @@ const MT5TradeDetailsItem: FC<TMT5TradeDetailsItemProps> = ({ className, label,
             <Text colorStyle='subtle' size='sm'>
                 {label}
             </Text>
-            <div className='flex items-center space-x-400 pr-400'>
+            <div className='flex items-center space-x-400'>
```

Comment:
If you are using `flex` use `gap` if not using `flex` use `space` 👍🏼 
```suggestion
            <div className='flex items-center gap-x-400'>
            <div className='space-x-400'>
```